scripts/fetchAndSubmitAddMemberEvents.js

```diff
@@ -0,0 +1,147 @@
+import dotenv from 'dotenv'
```

Comment:
Given how critical this migration process is, it's probably worth adding some comments here to explain what it's doing.

Reply:
Added some comments